feed.go

```diff
@@ -184,6 +184,25 @@ func (f *baseFeed) RemoveActivityByForeignID(foreignId string) error {
 	})
 }
 
+func (f *baseFeed) UpdateActivities(activities ...*Activity) error {
+	for _, a := range activities {
```

Comment:
we already do this check server-side

Reply:
Yup, realised it while working on the last PR, moving this away